File: m/rls/stlen.r
file	stlen - string length
include	rid:rider
include	rid:stdef

code	st_len - get string length

  func	st_len
	str : * Char			; string pointer
	()  : int			; string length
  is	bas : * Char			; base address
	bas = str			; remember it
	.. while *str++			; skip to end
	reply str - bas - 1		; return length
  end


File: m/rls/rlb/stlen.c
/* file -  stlen - string length */
#include "m:\rid\rider.h"
#include "m:\rid\stdef.h"
/* code -  st_len - get string length */
int st_len(
register char *str )
{ register char *bas ;
  bas = str;
  while ( *str++) { }
  return ( str - bas - 1);
} 
